examples/psyad/lfric/tangent_linear/Makefile

```diff
@@ -47,31 +47,46 @@ TL_KERNEL_NAME ?=
 TL_KERNEL_FILE = ${TL_KERNEL_NAME}_mod.F90
 ADJ_KERNEL_FILE = $(addprefix adj_,$(subst tl_,,${TL_KERNEL_FILE}))
 
-GENERATED_FILES = ${OUT_DIR}/adj*.[fF]90
+GENERATED_FILES = ${OUT_DIR}/adj*.[fF]90 ../adjoint/adj_transpose_matrix_vector_kernel_mod.F90 ../adjoint/adj_hydrostatic_kernel_mod.F90 ../adjoint/adj_moist_dyn_mass_kernel_mod.F90 ../adjoint/adj_w3_advective_update_kernel_mod.F90 ../adjoint/adj_combine_w2_field_kernel_mod.F90 ../adjoint/adj_split_w2_field_kernel_mod.F90 ../adjoint/adj_sample_flux_kernel_mod.F90 ../adjoint/adj_tracer_viscosity_kernel_mod.F90 ../adjoint/adj_dg_matrix_vector_kernel_mod.F90 ../adjoint/adj_w2_to_w1_projection_kernel_mod.F90
```

Comment:
No, because these are the files that now produce correct adjoint code from tangent-linear code so can go straight into the adjoint directory, however, there are other files in the adjoint directory that have had manual changes.